app/webpacker/components/Panel/pages/BannedCompetitorsPage/BannedCompetitors.jsx

```diff
@@ -1,67 +1,78 @@
-import React, { useState } from 'react';
-import {
-  Button, Icon, Modal, Table,
-} from 'semantic-ui-react';
-import UserBadge from '../../../UserBadge';
-import BanendCompetitorForm from './BannedCompetitorForm';
+import React, { useState } from "react";
+import { Button, Icon, Modal, Table, Segment } from "semantic-ui-react";
+import UserBadge from "../../../UserBadge";
+import BannedCompetitorForm from "./BannedCompetitorForm";
 
 export default function BannedCompetitors({
   bannedCompetitorRoles,
   sync,
   canEditBannedCompetitors,
+  isPastBannedCompetitorList,
 }) {
   const [banModalParams, setBanModalParams] = useState(null);
 
   return (
     <>
-      <Table>
-        <Table.Header>
-          <Table.Row>
-            <Table.HeaderCell width={5}>User</Table.HeaderCell>
-            <Table.HeaderCell width={2}>Start date</Table.HeaderCell>
-            <Table.HeaderCell width={2}>End date</Table.HeaderCell>
-            <Table.HeaderCell width={4}>Reason</Table.HeaderCell>
-            <Table.HeaderCell width={3}>Scope</Table.HeaderCell>
-            {canEditBannedCompetitors && <Table.HeaderCell width={2}>Edit</Table.HeaderCell>}
-          </Table.Row>
-        </Table.Header>
-
-        <Table.Body>
-          {bannedCompetitorRoles.map((role) => (
-            <Table.Row key={role.id}>
-              <Table.Cell>
-                <UserBadge
-                  user={role.user}
-                  hideBorder
-                  leftAlign
-                  subtexts={role.user.wca_id ? [role.user.wca_id] : []}
-                />
-              </Table.Cell>
-              <Table.Cell>{role.start_date}</Table.Cell>
-              <Table.Cell>{role.end_date}</Table.Cell>
-              <Table.Cell>{role.metadata.ban_reason}</Table.Cell>
-              <Table.Cell>{role.metadata.scope}</Table.Cell>
+      {canEditBannedCompetitors && !isPastBannedCompetitorList && (
+        <Button onClick={() => setBanModalParams({ action: "new" })}>
+          Ban new competitor
+        </Button>
+      )}
```

Comment:
This button can be moved to the parent component (BanendCompetitorsPage/index.jsx). This way, you don't have to pass the `isPastBannedCompetitorList` boolean.

The main challenge of moving this to parent component will be that you might have to move the modal and related states as well, and then pass the `editBannedCompetitor` action as a parameter to this component, and you can call that parameter as action for the edit.

Hope this is clear, let me know if you need more clarifications.

Reply:
@danieljames-dj I have moved the ban new competitor button and it's modal to the parent comp.  For linting,  I am running `npx eslint /path_to_file/file.jsx` but is asking to install eslint. Is that fine to insall/update packages and push code? I have not run it

Reply:
Thanks a lot @darshanvss. It's fine to install eslint I guess. It's up to you. Anyway, the current changes looks great to me.

Reply:
Re eslint: The issue is the package manager. For some reason, your environment wants to run `npx` which is a script executor for NodeJS. However, we are using **Yarn** to install and link our packages in JS, so you should run eslint through Yarn as well.